### PR TITLE
Add latest PHP releases: 7.0.21 and 7.1.7

### PR DESCRIPTION
**Pull request by ottlinger**
As I've learned via https://github.com/travis-ci/travis-ci/issues/8054 it's best to add new releases here, so please add:
* 7.0.21
* 7.1.7

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Thanks for the PR. Please also update the other test file, the change log, and add the new php 7.2 version too. Then I'll merge. :)

**Comment by ottlinger** [comment]
@GrahamCampbell not sure if I got all of your proposed changes .... did I miss something ?! Thanks.

File: share/php-build/definitions/7.2.0alpha3 [hunk 1]
```diff
@@ -0,0 +1,3 @@
+install_package "https://downloads.php.net/~pollita/php-7.2.0alpha2.tar.bz2"
```
Comment (GrahamCampbell) [inline share/php-build/definitions/7.2.0alpha3]: typo

**Comment by GrahamCampbell** [comment]
Please also update https://github.com/php-build/php-build/blob/master/run-tests.sh.

**Comment by Majkl578** [comment]
I think you will need to disable XDebug for now since it was broken between alpha2 & alpha3: https://github.com/php/php-src/commit/caaeb4849aa56cbbdc66ea015c11a58bd47a43ff & https://bugs.xdebug.org/view.php?id=1452

**Comment by GrahamCampbell** [comment]
Thanks. Will address xedbug later.

**Pull request merged**